### ISSUE TITLE
Kill Gnuplot 

### ISSUE THREAD

**Issue by apetros** [realgjl/sfcNordic#3]
Is the kill Gnuplot command needed inside the function end_simulatio? I think that if the Call_GP is set to F in the settings, you do not need to do this. Please check and remove it from the code to be cleaner.

**Comment by albertjlguo** [comment]
Yes, the kill Gnuplot command is unnecessary if we have call_gp to False and we want to generate many simulations.
But sometimes we may need to check the plot directly via gnuplot.
Plus, a kill gnuplot command has no side effects.
Thus I think keeping it better than removing it.

**Comment by apetros** [comment]
OK. However, the code used is not portable. It only works in Windows (while RAMSES works both in windows and Linux).
